R/read_servers.R

```diff
@@ -34,29 +34,38 @@
 #' @examples
 #' \dontrun{
 #' data <- sql_server_read_data(
-#'   user          = "rfamro",
-#'   password      = "",
+#'   dbms          = "MySQL",
+#'   driver_name   = "",
 #'   host          = "mysql-rfam-public.ebi.ac.uk",
-#'   port          = 4497,
 #'   database_name = "Rfam",
+#'   user_name     = "rfamro",
+#'   password      = "",
+#'   port          = 4497,
 #'   src           = "author",
-#'   driver_name   = "",
-#'   dbms          = "MySQL"
+#'   records       = NULL,
+#'   fields        = NULL,
+#'   id_position   = NULL,
+#'   id_col_name   = NULL
 #' )
 #' }
 #' @importFrom magrittr %>%
 #' @keywords internal
 #' @noRd
-sql_server_read_data <- function(user = "rfamro", password = "",
-                                 host = "mysql-rfam-public.ebi.ac.uk",
-                                 port = 1433L,
-                                 database_name = "Rfam", driver_name = "",
-                                 src = NULL, records = NULL,
-                                 fields = NULL, id_position = NULL,
-                                 id_col_name = NULL, dbms = "MySQL") {
+sql_server_read_data <- function(dbms,
+                                 driver_name,
+                                 host,
```

Comment:
This should be `base_url`. I understand DBI does not expect a URL but a hostname but the transformation should be done internally. The main value of readepi is that it provides a **unified** interface to the different packages so let's put emphasis on this.

Reply:
Also reorder args to align with other functions please

Reply:
this has been accounted for at commit `aa321015`